initramfs-init.in

```diff
@@ -646,6 +652,19 @@ if [ -n "$ovlfiles" ]; then
 else
 	apk add --root $sysroot $repo_opt $apkflags $pkgs
 fi
+
+if [ -n "$s390x_installer" ]; then
+	rc_add sshd default
+	echo "PermitRootLogin yes" >> $sysroot/etc/ssh/sshd_config
+	# to make modloop happy
+	mkdir -p $sysroot/.modloop/modules/$(uname -r)
+	if [ -n "${s390x_installer#*,}" ]; then
+		echo "root:${s390x_installer#*,}" > $sysroot/.passwd
+		chroot $sysroot sh -c 'chpasswd < /.passwd'
+		rm $sysroot/.passwd
+	fi
+fi
+
```

Comment:
As far I can see, there is nothing in this that is specific for s390x. Would it be an idea to rename the `s390x_installer` variable to `ssh_install` or similar?